src/pages/workspace/WorkspacesListRow.tsx

```diff
@@ -163,11 +166,23 @@ function WorkspacesListRow({
                 </View>
             </View>
             {isWide && (
-                <ThreeDotsMenu
-                    menuItems={menuItems}
-                    anchorPosition={{top: 0, right: 0}}
-                    iconStyles={[styles.mr2]}
-                />
+                <View ref={threeDotsMenuContainerRef}>
+                    <ThreeDotsMenu
+                        onIconPress={() => {
+                            threeDotsMenuContainerRef.current?.measureInWindow((x, y, width, height) => {
+                                setThreeDotsMenuPosition({
+                                    horizontal: x,
+                                    vertical: y + height + variables.componentBorderRadiusSmall,
```

Comment:
The use of `variables.componentBorderRadiusSmall` feels a little workaround-ish. The style can change and we can forget to change this calculation. But how is the border radius affecting the vertical position of the popover?

Reply:
cc: @MaciejSWM

Reply:
Fixed by changing anchorAlignment to `horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT` and then didn't need that additional offset